src/test-data/localizationData.csv

```diff
@@ -0,0 +1,4 @@
+languageName,url,headerKeywords,footerKeywords
```

Comment:
I would recomend to use JS/JSON to store language mapping as they natively supported by JavaScript